lib/rules/no-forbidden-elements.js

```diff
@@ -32,6 +33,17 @@ function sanitizeConfigArray(forbidden = []) {
   return forbidden.filter(Boolean).sort((a, b) => b.length - a.length);
 }
 
+function hasHeadParent(path) {
+  let parents = [...path.parents()];
+  let refParentNode = {
+    type: 'ElementNode',
+    tag: 'head',
+  };
+  let hasHeadElementInParentPath = parents.find((parent) =>
```

Comment:
Would using `some` here make more sense?

Reply:
good call, changed it.